routers/web/user/task.go

```diff
@@ -28,6 +30,27 @@ func TaskStatus(ctx *context.Context) {
 		return
 	}
 
+	if err := task.LoadRepo(ctx); err != nil {
+		ctx.JSON(http.StatusInternalServerError, map[string]any{
+			"err": err,
```

Comment:
Maybe @techknowlogick is interested in this line (prevent from exposing internal messages to end users, it might contain sensitive information like internal IP/Port, etc)

Reply:
8d067d814114b928fff969f3cca2ac49f8be0968 I changed the code to avoid leaking the real reason to frontend.